server/commands.py

```diff
@@ -617,3 +631,55 @@ def ooc_cmd_mutepm(client, arg):
         client.send_host_message('You stopped receiving PMs')
     else:
         client.send_host_message('You are now receiving PMs')
+
+def ooc_cmd_ban(client, arg):
+    if not client.is_mod:
+        raise ClientError('You must be authorized to do that.')
+    ip = arg.strip()
+    if len(ip) < 7:
```

Comment:
Where is this magical 7 from? If you want to check for a valid IP address, you should at least do it right. IPv6 addresses are quite tricky.